givenergy_modbus/model/inverter.py

```diff
@@ -312,6 +347,12 @@ class InverterRegisterGetter(RegisterGetter):
         "temp_battery": Def(C.deci, None, IR(56)),
         "i_grid_port": Def(C.centi, None, IR(58)),
         "battery_percent": Def(C.uint16, None, IR(59)),
+        "e_battery_discharge_total": Def(C.deci, None, IR(105)),
```

Comment:
... But perhaps you added them here by mistake ?
105 and 106 overlap the battery input register block 60-119.